### ISSUE TITLE
Is is possible to get access to the admin panel http://127.0.0.1:8000/admin/

### ISSUE THREAD

**Issue by Mike-in-Maine** [app-generator/django-volt-dashboard#6]
Once the template is installed, is it possible to gain access to the admin panel?

**Comment by app-generator** [comment]
Hello @Mike-in-Maine 

You need to create a superuser via:

```bash
$ python manage createsuperuser
```

**Comment by Mike-in-Maine** [comment]
Ok thank you . So I would like to report a little bug here: I knew I had to use '$ python manage.py createsuperuser', however IF you already created a new user through http://127.0.0.1:8000/login, superuser cannot be the same as your login, and while creating superuser, it will not tell you that this operation cannot be accomplished.
It does make sense since you cannot have 2 users with the same name, its just something that is not immediately clear.

I solved it by creating a different user and pass for superuser than the user I use for http://127.0.0.1:8000/login. --Thank you!
